cirq-core/cirq/ops/clifford_gate.py

```diff
@@ -143,10 +143,6 @@ def _gate_tableau(num_qubits: int, gate: raw_types.Gate) -> 'cirq.CliffordTablea
 class CommonCliffordGateMetaClass(value.ABCMetaImplementAnyOneOf):
     """A metaclass used to lazy initialize several common Clifford Gate as class attributes."""
 
-    # These are class properties so we define them as properties on a metaclass.
-    # Note that in python 3.9+ @classmethod can be used with @property, so these
-    # can be moved to CommonCliffordGates.
```

Comment:
This ability was removed again in 3.11, so deleting the comment. https://stackoverflow.com/questions/128573/using-property-on-classmethods/64738850#64738850